### PR TITLE
Fix #1025: Removed getTopic and getStory

### LINKED ISSUE #1025: Remove getStory and getTopic from domain layer
There are two different types of functions for `getTopic` and `getStory`, we do not need the function which does not contain profile id.

Remove/Update the test cases also.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #1025 

There are two different types of functions for getTopic and getStory, we do not need the function which does not contain profile id.

Remove/Update the test cases also.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt [hunk 1]
```diff
@@ -90,8 +90,10 @@ val TOPIC_FILE_ASSOCIATIONS = mapOf(
 
 private const val QUESTION_DATA_PROVIDER_ID = "QuestionDataProvider"
```
Comment (veena14cs) [inline domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: nit change: remove this blank line

**Review by veena14cs — APPROVED** [review]
LGTM.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt (hunk 1 above)
Comment (rt4914) [reply domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: Done.

**Pull request merged**